### ISSUE TITLE
Increasing mobile and search friendliness

### ISSUE THREAD

**Issue by piranha** [opengaming/osgameclones#1077]
Hello! I think ultimately the project goal is providing information to people about various games they can play and contribute to.

To me this means that ranking higher in Google is a good thing: site can be seen by more people, thus converting more of them to players/contributors to games/contributors to site. :)

There is a problem with our single page design, though. While it's been great and serving well, it's now over 2 Mb in size. This means Google thinks we are slow. Incidentally, I think it's also a PITA for mobile devices - and Google reports we're not quite great for them both [speed-wise](https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fosgameclones.com%2F) and design-wise:

![image](https://user-images.githubusercontent.com/6553/77325637-51168480-6d21-11ea-84d5-8b3323dbe862.png)

Any thoughts?

**Comment by cxong** [comment]
I think we will need a mobile version of the site; some features that work well for desktop like clickable tag filters will need to work differently for mobile

**Comment by piranha** [comment]
Yeah, they need to be a little bit bigger, so it's easier to tap with a finger.

I'm a little bit concerned about page size as well, being over 2 mb is a little bit excessive, what'd you think? :)
